BraveRewardsUI/Settings/Ads/Details/AdsDetailsViewController.swift

```diff
@@ -82,14 +96,73 @@ class AdsDetailsViewController: UIViewController {
     Strings.fourAdsPerHour,
     Strings.fiveAdsPerHour
   ]
-  
+
+  // TODO(https://github.com/brave/brave-browser/issues/10316) return subdivisions from ads lib
```

Comment:
Any work left on this? If not we can make a follow up ticket in brave-ios

Reply:
Doesn't look like it based on that ticket (still open). @tmancey can maybe comment better

Reply:
That was mostly a comment from me wondering why each client was defining this list instead of the ads library, so its not really an issue

Reply:
https://github.com/brave/brave-browser/issues/10316 is not required for now, and will be triaged at a later date. Thanks